PyInstaller/utils/win32/versioninfo.py

```diff
@@ -294,81 +298,64 @@ def __init__(self, filevers=(0, 0, 0, 0), prodvers=(0, 0, 0, 0),
+        return struct.pack(
+            '13L', self.sig, self.strucVersion, self.fileVersionMS, self.fileVersionLS, self.productVersionMS,
+            self.productVersionLS, self.fileFlagsMask, self.fileFlags, self.fileOS, self.fileType, self.fileSubtype,
+            self.fileDateMS, self.fileDateLS
+        )
 
     def __eq__(self, other):
         return self.toRaw() == other
 
     def __str__(self, indent=u''):
-        fv = (self.fileVersionMS >> 16, self.fileVersionMS & 0xffff,
-              self.fileVersionLS >> 16, self.fileVersionLS & 0xFFFF)
-        pv = (self.productVersionMS >> 16, self.productVersionMS & 0xffff,
-              self.productVersionLS >> 16, self.productVersionLS & 0xFFFF)
+        fv = (
+            self.fileVersionMS >> 16, self.fileVersionMS & 0xffff, self.fileVersionLS >> 16, self.fileVersionLS & 0xFFFF
+        )
+        pv = (
+            self.productVersionMS >> 16, self.productVersionMS & 0xffff, self.productVersionLS >> 16,
+            self.productVersionLS & 0xFFFF
+        )
         fd = (self.fileDateMS, self.fileDateLS)
-        tmp = [u'FixedFileInfo(',
-            u'# filevers and prodvers should be always a tuple with four items: (1, 2, 3, 4)',
+        tmp = [
+            u'FixedFileInfo(', u'# filevers and prodvers should be always a tuple with four items: (1, 2, 3, 4)',
             u'# Set not needed items to zero 0.',
             u'filevers=%s,' % (fv,),
-            u'prodvers=%s,' % (pv,),
-            u"# Contains a bitmask that specifies the valid bits 'flags'r",
+            u'prodvers=%s,' % (pv,), u"# Contains a bitmask that specifies the valid bits 'flags'r",
             u'mask=%s,' % hex(self.fileFlagsMask),
             u'# Contains a bitmask that specifies the Boolean attributes of the file.',
-            u'flags=%s,' % hex(self.fileFlags),
-            u'# The operating system for which this file was designed.',
+            u'flags=%s,' % hex(self.fileFlags), u'# The operating system for which this file was designed.',
             u'# 0x4 - NT and there is no need to change it.',
-            u'OS=%s,' % hex(self.fileOS),
-            u'# The general type of file.',
-            u'# 0x1 - the file is an application.',
-            u'fileType=%s,' % hex(self.fileType),
-            u'# The function of the file.',
+            u'OS=%s,' % hex(self.fileOS), u'# The general type of file.', u'# 0x1 - the file is an application.',
+            u'fileType=%s,' % hex(self.fileType), u'# The function of the file.',
             u'# 0x0 - the function is not defined for this fileType',
-            u'subtype=%s,' % hex(self.fileSubtype),
-            u'# Creation date and time stamp.',
-            u'date=%s' % (fd,),
-            u')'
+            u'subtype=%s,' % hex(self.fileSubtype), u'# Creation date and time stamp.',
+            u'date=%s' % (fd,), u')'
         ]
-        return (u'\n'+indent+u'  ').join(tmp)
+        return (u'\n' + indent + u'  ').join(tmp)
```

Comment:
This looks messy. We should keep one entry per line. (Plus, remove the 'u' prefix).